library/src/main/java/hotchemi/android/rate/AppRate.java

```diff
@@ -69,8 +64,8 @@ public AppRate setRemindInterval(int remindInterval) {
         return this;
     }
 
-    public AppRate setShowNeutralButton(boolean isShowNeutralButton) {
-        this.isShowNeutralButton = isShowNeutralButton;
+    public AppRate setShowLaterButton(boolean isShowNeutralButton) {
+        options.setShowNeutralButton( isShowNeutralButton );
```

Comment:
Can you fix formatter setting to the default of android studio? I don't like it.